docker/README.md

```diff
@@ -13,6 +13,23 @@ of this directory in the past releases for more info.
 For help please try [Fedimint Github Discussions](https://github.com/fedimint/fedimint/discussions)
 or `#mint-ops` channel on [Fedimint's Discord server](https://chat.fedimint.org/).
 
+## System Requirements
+### 3-of-4 Federation with remote Bitcoin backend
+* Memory: 500MB minimum (activate swap in that case), 1GB recommended
+* Disk: 10GB minimum
+* CPU: 1 core minimum, 2 cores recommended
+
+Notably, we did successfully test a 3-of-4 federation with one guardian on a Raspberry Pi Zero 2 W, which is likely the
+lowest power device one can run Fedimint on.
+
+### 3-of-4 Federation with local Bitcoin backend
+* Memory: 2GB minimum, 4GB recommended
+* Disk: 1TB recommended (50GB if running pruned, but there be dragons)
+* CPU: 2 cores minimum, 4 cores recommended
+
+Worst-case memory consumption scales with the number of guardians (about 50MB per guardian), larger federations may be
```

Comment:
@joschisan is this correct? I think you mentioned the 50MB number on one of our recent calls. I thought it was larger, but I might have been mistaken.